sekoia-io-xdr/list_alerts.py

```diff
@@ -8,13 +8,20 @@
 
 def list_alerts(config, params):
     url: str = OPERATION_CENTER_BASE_URL
+        
+    try:
+        created_at = params.get("created_at")
+        created_at = (datetime(created_at.split(",")[0]), datetime(created_at.split(",")[1]))
```

Comment:
Are you sure about splitting the created_at expression in tuple? How request behaves with a tuple in the parameters?

```suggestion
        created_at = (datetime(created_at.split(",")[0]), datetime(created_at.split(",")[1]))
```